tests/data/SPECS/test-subpackages-epoch.spec

```diff
@@ -0,0 +1,49 @@
+Name:           test
+Epoch:          1
+Version:        1.0
+Release:        1
+Summary:        Test
+
+License:        Public Domain
+URL:            https://fedoraproject.org
+Source:         hello.c
+
+%description
+%{summary}.
+
+%package test2
+Summary: Test2.
+%description test2
+
+%package test3
+Epoch:   0
+Summary: Test3.
+%description test3
+
+%prep
+%autosetup -c -D -T
+cp -a %{S:0} .
+
+%build
+gcc hello.c -o hello
+cp hello.c hello2.c
+gcc hello2.c -o hello2
+cp hello.c hello3.c
+gcc hello3.c -o hello3
```

Comment:
We're not interested in any compilation results here, don't waste time with stuff that is not actually used for anything. It may not seem much but it's a death by thousand cuts, every single redundant action in the test-suite hurts. This test-package is not concerned with files at all so don't put any in there. Which means you can ditch %prep, %build and %install.

Reply:
Of course, if you just use rpmspec for the test as I suggested, it doesn't actually matter as much, but it does help to keep reproducers to bare minimum in any case.